filesystem/src/main/java/org/frankframework/filesystem/FileSystemListener.java

```diff
@@ -529,14 +526,6 @@ public void setOverwrite(boolean overwrite) {
 		this.overwrite = overwrite;
 	}
 
-	/**
-	 * Determines the contents of the message that is sent to the pipeline. Can be 'name', for the filename, 'path', for the full file path, 'contents' for the contents of the file, 'info' for file information. For any other value, the attributes of the file are searched and used
-	 * @ff.default path
-	 */
-	public void setMessageType(String messageType) {
-		this.messageType = messageType;
-	}
-
```

Comment:
Deze moet je wel laten staan (met enum param) zodat de JavaDoc in de FrankDoc komt :)

Reply:
Ik heb deze verwijderd, omdat deze class geërfd wordt door twee soorten listeners die beide een andere enum nodig hebben (MailListener vs Directory, (s)Ftp en Samba Listeners). Deze functie laten staan kan niet, omdat de children de signature van de functie moeten aanpassen (omdat het 2 verschillende enums zijn). Vandaar dat ik deze functie naar de children heb gekopieerd.